### DISCUSSION TITLE
Bug in wizard next step when using multiple wizards inside tabs

### DISCUSSION THREAD

**Discussion by AHMED-GAMAL-AG** [filamentphp/filament#12828] [Help]
### Package

Form builder

### Package Version

3.2

### How can we help you?

When using tabs and having a wizard in each tab, when I click the next action, I get the next step in the second wizard, not in the one I'm using

for example, in the first tab, there is a wizard with 3 steps When I click next, it goes to step number 2 for this wizard and also step number 1 in the second tab with the other wizard

### video demonstration

https://github.com/filamentphp/filament/assets/76778937/beb108dc-54f2-428a-bbbb-f9943cd7f8a5

is this problem a bug or there is something going wrong here ?

**Comment by leandrocfe** [comment]
Try to add an id to the wizard `->id('xx')`
